shell/platform/darwin/macos/framework/Source/FlutterBackingStoreData.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterFrameBufferProvider.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterIOSurfaceHolder.h"
+
+#import <Cocoa/Cocoa.h>
+
+/**
+ * FlutterBackingStoreData holds data to be stored in the
+ * BackingStore's user_data.
+ */
+@interface FlutterBackingStoreData : NSObject
+
+- (nullable instancetype)initWithLayerId:(size_t)layerId
+                              fbProvider:(nonnull FlutterFrameBufferProvider*)fbProvider
+                         ioSurfaceHolder:(nonnull FlutterIOSurfaceHolder*)ioSurfaceHolder;
+
+- (size_t)getLayerId;
+- (nonnull FlutterFrameBufferProvider*)getFrameBufferProvider;
+- (nonnull FlutterIOSurfaceHolder*)getIOSurfaceHolder;
+
```

Comment:
These could be properties. Also `get*` is not obj-c style. These could be:

```
@property(readonly, nonatomic, nonnull) FlutterFrameBufferProvider* frameBufferProvider;
```

etc.

Reply:
Done